CONTRIBUTING.md

```diff
@@ -1,3 +1,36 @@
 # Contributing to Twilio
 
 All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.
+
+## Contributing to this repo
+
+This is a monorepo consisting of 5 packages. To setup to develop with the packages, follow these steps:
+
+### Setting up the monorepo
+
+Clone the repo:
+
+```bash
+git clone https://github.com/twilio-labs/twilio-style.git
+cd twilio-style
+```
+
+Install the monorepo dependencies:
+
+```bash
+npm install
+```
+
+Bootstrap the monorepo:
```

Comment:
Can you remove this? We have a `postinstall` command that runs this https://github.com/twilio-labs/twilio-style/blob/main/package.json#L31